src/json.c

```diff
@@ -33,29 +33,29 @@ along with GNU Emacs.  If not, see <https://www.gnu.org/licenses/>.  */
 #define JSON_HAS_ERROR_CODE (JANSSON_VERSION_HEX >= 0x020B00)
 
 #ifdef WINDOWSNT
-# include <windows.h>
-# include "w32common.h"
-# include "w32.h"
+#include <windows.h>
+#include "w32.h"
+#include "w32common.h"
 
 DEF_DLL_FN (void, json_set_alloc_funcs,
-	    (json_malloc_t malloc_fn, json_free_t free_fn));
-DEF_DLL_FN (void, json_delete, (json_t *json));
-DEF_DLL_FN (json_t *, json_array, (void));
-DEF_DLL_FN (int, json_array_append_new, (json_t *array, json_t *value));
+            (json_malloc_t malloc_fn, json_free_t free_fn));
```

Comment:
Don't make these whitespace changes in C files; it'll just make it harder to merge later.